news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: post
+title: "The Followpoint: Hivie, the Man of Anything and Everything"
+date: 2024-11-25 17:00:00 +0000
+series: The Followpoint
+---
+
+Ever heard of the person who does literally anything you can think of? No? Well, in this The Followpoint anniversary interview you'll learn more about such a person!
+
+![](/wiki/shared/news/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a series where we interview various players and community figures to get to know them better! (Note: Please pretend this news post released on November 10) Today is a special day for us, it's our first anniversary! And we just had to chose a special interviewee for today, both for us and osu! community in general, [Hivie](https://osu.ppy.sh/users/14102976)!
+
+Hivie is an [osu!team](/wiki/People/osu!_team) member responsible for community affairs, [NAT (Nomination Assessment Team)](/wiki/People/Nomination_Assessment_Team) leader, [Project Loved](/wiki/Community/Project_Loved) manager, [medals](/wiki/Medals) developer and works on even more things than I listed here, so you likely heard his name at least once, no matter the aspect of osu! you are interested in! Funny enough, [our first interview](https://osu.ppy.sh/home/news/2023-11-10-the-followpoint-ephemeral-the-man-behind-the-medals) was with [Ephemeral](https://osu.ppy.sh/users/102335), who also was a community maintainer and medals developer. That, and the fact Hivie often helps us with polishing our interviews makes him a perfect interviewee for today! 
+
+This interview will looks slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  
```

Comment:
```suggestion
This interview will look slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  
```